rendercv/renderer/renderer.py

```diff
@@ -12,7 +12,6 @@
 from typing import Optional
 
 import fitz
```

Comment:
Make sure to use

```
hatch run default:sort-imports
```

and

```
hatch run default:format
```

See the developer guide for details.

Reply:
When I run:

```bash
hatch run default:format
```

It generates back to this format. May be there are some problem with my configuration? I'm not being able to figure this out.